test/e2e/fixtures.go

```diff
@@ -300,10 +303,45 @@ func setupTestForFlowAggregator(tb testing.TB, o flowVisibilityTestOptions) (*Te
 	if err := testData.deployFlowAggregator(ipfixCollectorAddr, o); err != nil {
 		return testData, v4Enabled, v6Enabled, err
 	}
+	tb.Logf("Checking Flow Exporters can resolve Flow Aggregator Service address correctly")
+	err = checkResolveAddress(testData)
+	if err != nil {
+		return testData, v4Enabled, v6Enabled, err
+	}
 
 	return testData, v4Enabled, v6Enabled, nil
 }
 
+func checkResolveAddress(data *TestData) error {
```

Comment:
that's really not ideal IMO (relying on logs for his). I wonder if we could update `antctl` for the FA to support printing a status (JSON and text format) with the following information:
* number of Agents connected
* connection to CH ready
* connection to IPFIX collector ready

Then in the e2e tests we would call that command and extract the data we need from the JSON output

That being said, we can merge this as it is to fix the tests, and handle that in a future PR

Reply:
We already have "antctl get recordmetrics" to check the number of Agents connected. We can extend it to check which collector we are connecting to. 

root@flow-aggregator-7799b785d4-p46cr:/# antctl get recordmetrics
RECORDS-EXPORTED RECORDS-RECEIVED FLOWS EXPORTERS-CONNECTED
12480            48951            7     3     

But why do we want to run antctl in the e2e test? I thought retrieving log from a Pod is easier?

Reply:
IIRC we have several other tests that use antctl to validate some conditions.
It's better than relying on a log message in my opinion:
- there is no contract that log messages will stay the same over time; I should not have to update an e2e test if I change a log message string (for unit tests, we can make an exception since the test cases live alongside the code). It is different for APIs or antctl.
- info log messages can be disabled, and I don't think it should impact our ability to run the e2e tests.

I suggested antctl, but if we can access the FA API directly to get this information, it's even better

Reply:
Additionally, looking for a signal in the logs can be misleading unless the Pod is restarted at the beginning of the test (we can get stale information more easily than with an API call).

Reply:
> we can get stale information more easily than with an API call

That is the main reason I want to restart antrea-agent Pods. I'll try to create a restful client to see if we can't access the FA API directly to check the number of Agents connected.

For `connection to CH ready` and `connection to IPFIX collector ready`, we can open another PR to extend these metrics in the API call.

Reply:
@antoninbas 
I've extended the recordmetrics endpoint in FA's API and used this metric to verify if the Flow Aggregator is successfully connected to the "IPFIX collector Pod"/"ClickHouse"/"Antrea-Agent Pods".